### ISSUE TITLE
Importing a baremetal server without an OS crashes terraform

### ISSUE THREAD

**Issue by monorkin** [scaleway/terraform-provider-scaleway#1829]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

```
Terraform v1.4.0
on linux_amd64
+ provider registry.terraform.io/scaleway/scaleway v2.13.0
```

### Affected Resource(s)

* scaleway_baremetal_server

### Terraform Configuration Files

```hcl
terraform {
  required_version = ">= 0.13"

  cloud {
    hostname = "app.terraform.io"
    organization = "<YOUR ORG>"

    workspaces {
      name = "<YOUR WORKSPACE>"
    }
  }

  required_providers {
    scaleway = {
      source = "scaleway/scaleway"
    }
  }
}

provider "scaleway" {
  zone = "fr-par-2"
  region = "fr-par"
}

variable "project_id" {
  type = string
  description = "Your project ID."
  default = "<YOUR PROJECT ID>"
}

resource "scaleway_account_ssh_key" "main" {
  name = "main"
  public_key = "<YOUR PUBLIC SSH KEY>"
}

resource "scaleway_baremetal_server" "web" {
  name = "web"
  hostname = "web"
  zone = "fr-par-2"
  offer = "EM-A315X-SSD"
  os = "96e5f0f2-d216-4de2-8a15-68730d877885"
  ssh_key_ids = [data.scaleway_account_ssh_key.main.id]
}
```

### Debug Output

Couldn't obtain one since I fixed my issue by installing an OS on the machine manually

### Panic Output

https://gist.github.com/monorkin/f88f8e193d98fd149ba035ae79031500

### Expected Behavior

The server should get imported or it should return a friendly error message. 

### Actual Behavior

Terraform crashes showing a stack trace

### Steps to Reproduce

1. `terraform import scaleway_baremetal_server.web fr-par-2/<YOUR SERVER ID>`

### Important Factoids

* Authentication with Scaleway is done through the `scw`config file
* After I manually installed an OS on the machine the import passed without issue

### References

* Might be related to #1746 and #1756

**Comment by remyleone** [comment]
Hello @monorkin this is issue should be fixed in the latest version v2.13.1. Let us know if you encounter another issue :)

**Comment by monorkin** [comment]
Thank you for the quick turnaround on this! Have a great weekend.
